src/components/core/newsList/List.js

```diff
@@ -42,9 +26,9 @@ const NewsList = (props) => {
           </tr>
         </thead>
         <tbody>
-          {posts.map((post) => (
-            <tr key={post.data.id}>
-              <td>{post.data.id}</td>
+          {posts.map((post, key) => (
+            <tr key={post.data.index}>
```

Comment:
Please search about the key prop. What is it ? & why it is mandatory to add when in loop.